app/services/publisher_statement_getter.rb

```diff
@@ -27,7 +27,7 @@ def earning_period
       if eyeshade_settlement? || fee?
         created_at.prev_month.at_beginning_of_month
```

Comment:
Should we also include `to_date` here?

Reply:
https://github.com/brave-intl/publishers/pull/2690/commits/ec797dc64d5f339470d54e9ede1bb264c81f2c52